sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image-bitmap-utils.js

```diff
@@ -52,9 +52,13 @@ function runOneIterationImageBitmapTest(useTexSubImage2D, bindingTarget, program
         break;
     }
 
-    debug('Testing ' + (useTexSubImage2D ? 'texSubImage2D' : 'texImage2D') +
-          ' with flipY=' + flipY + ' and premultiplyAlpha=' + premultiplyAlpha +
-          ', bindingTarget=' + (bindingTarget == gl.TEXTURE_2D ? 'TEXTURE_2D' : 'TEXTURE_CUBE_MAP'));
+    if (!optionsVal.is3D)
```

Comment:
The CL looks good.  Then you also need to update all includes of the utils.js and remove "2d".